features/step_definitions/machine_misc.rb

```diff
@@ -0,0 +1,20 @@
+Given(/^version dependent csi feature$/) do 
```

Comment:
@miyadav the wording for this step is confusing to me when I read it.  Can you perhaps reword the sentence so it's more descriptive of the purpose of the step?